static/css/shot-index.scss

```diff
@@ -252,6 +258,17 @@ h1 {
   }
 }
 
+.delete-confirmation-dialog {
```

Comment:
Maybe the default dialog style could be declared in one spot, and overridden in the other, to minimize duplication? Not sure if it's worth it, your call

Reply:
Since the dialog is absolute positioned to the delete button's parent, the defaults that work for one page would need to be completely overridden for the other page anyway, so I left them out of the partial.

Reply:
Right, that makes sense.